server/app.ts

```diff
@@ -1,12 +1,98 @@
-import express from 'express';
+import express from "express";
+import { InfluxDB } from "@influxdata/influxdb-client";
 
-export const app = express()
-const port = 3000
+// These are values that we provide from our InfluxDB instance.
+// In the real project, these should go in a .env file!
+const token =
+  "kxF9oWgVQdrUMz3kFOwyVw8eRqr1guWFeuSVIdXlBi-8ANuuwH5utNX-bDqh1hbhbmQguXYnPNJCr7AwUONtVA==";
+const org = "codebase-relativity";
+const bucket = "relativity-ramp-up";
+const url = "https://us-west-2-1.aws.cloud2.influxdata.com";
 
-app.get('/', (req, res) => {
-  res.send('Hello World!')
-})
+const client = new InfluxDB({ url, token });
+const queryApi = client.getQueryApi(org);
+export const app = express();
+const port = 8000;
+
+app.get("/buckets", async (req, res) => {
+  try {
+    const fluxQuery = "buckets()";
+    res.setHeader("Access-Control-Allow-Origin", "http://localhost:3000");
+    res.json(await queryApi.collectRows(fluxQuery));
+  } catch (error) {
+    console.log(error);
+  }
+});
+
+// Measurements!
+// - engine-1
+// - engine-2
+// - engine-3
+// - altitude
+// - speed
+
+// time: 2021-02-05T21:18:29.690Z,
+// measurement: speed,
+// flight (tag): 1,
+// value (field): 2420.76
+
+const relativityBucket = 'from(bucket: "relativity-ramp-up")';
 
 app.listen(port, () => {
-  console.log(`Example app listening at http://localhost:${port}`)
-})
+  console.log(`Example app listening at http://localhost:${port}`);
+});
+
+//get altitude data
+app.get("/altitude", async (req, res) => {
+  try {
+    const fluxQuery =
+      relativityBucket +
+      "|> range(start: 2021-02-05T21:13:29.690Z, stop: 2021-02-05T21:18:29.690Z)" +
+      "|> filter(fn: (r) =>" +
+      'r._measurement == "altitude")';
+    res.setHeader("Access-Control-Allow-Origin", "http://localhost:3000");
```

Comment:
nice job figuring this out! there are libs that we'll use that let us do this automatically and saves us a few lines of code